pwa/src/templates/templateParts/logsTable/LogsTableTemplate.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from "react";
+import * as styles from "./LogsTableTemplate.module.css";
+
+import _ from "lodash";
+import { t } from "i18next";
+import { navigate } from "gatsby";
+import { Tag, ToolTip } from "@conduction/components";
+import { Button } from "@gemeente-denhaag/components-react";
+import { faArrowRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Table, TableHead, TableRow, TableHeader, TableBody, TableCell } from "@gemeente-denhaag/table";
+import { Paginate } from "../../../components/paginate/Paginate";
+
+interface LogsTableTemplateProps {
+  logs: any[];
+  pagination: {
+    totalPages: number;
+    currentPage: number;
+    changePage: React.Dispatch<React.SetStateAction<number>>;
+  };
+}
+
+export const LogsTableTemplate: React.FC<LogsTableTemplateProps> = ({ logs, pagination }) => {
+  const handleResourceClick = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>,
+    path: string,
+    id: string,
+  ) => {
+    e.stopPropagation();
+
+    navigate(`/${path}/${id}`);
+  };
+
+  return (
+    <div className={styles.container}>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableHeader>{t("Level")}</TableHeader>
+            <TableHeader>{t("Message")}</TableHeader>
+            <TableHeader>{t("Endpoint")}</TableHeader>
+            <TableHeader>{t("Schema")}</TableHeader>
+            <TableHeader>{t("Cronjob")}</TableHeader>
+            <TableHeader>{t("Action")}</TableHeader>
+            <TableHeader>{t("User")}</TableHeader>
+            <TableHeader>{t("Organization")}</TableHeader>
+            <TableHeader>{t("Application")}</TableHeader>
+          </TableRow>
+        </TableHead>
+
+        <TableBody>
+          {logs.map((log: any) => (
+            <TableRow className={styles.tableRow} onClick={() => navigate(`/logs/${log._id.$oid}`)} key={log._id.$oid}>
+              <TableCell>
+                <Tag layoutClassName={styles[log.level_name]} label={_.upperFirst(_.lowerCase(log.level_name))} />
+              </TableCell>
+
+              <TableCell>
+                <ToolTip tooltip={log.message}>
+                  <div className={styles.message}>{log.message}</div>
+                </ToolTip>
```

Comment:
I think we should set a limit to the tooltip

![image](https://user-images.githubusercontent.com/43807324/217210305-4e879f88-733f-4e5f-a30f-bbccfd07a90c.png)